src/export/smtlib.ml

```diff
@@ -0,0 +1,105 @@
+(* This module contains the pretty-printer functions for smtlib *)
+
+open Dolmen_std
+
+open Term
+
+(* auxiliar functions *)
+let pp_builtins = function
```

Comment:
Maybe reserve the `pp_*` names for functions that actually print things (and thus have a type `Format.formatter -> 'a -> unit`). Maybe use a name like `pretty_builtins` ?

Reply:
Yes. It's that the idea.